### DISCUSSION TITLE
Esun and creality profiles

### DISCUSSION THREAD

**Discussion by nickyt1** [OrcaSlicer/OrcaSlicer#1559] [General]
Hi could we get some esun fillament profiles please, also can i get the creality cr10 s pro profile added please.

**Comment by ewilts** [comment]
For the Bambu X1C, there are eSun filament profiles at https://www.printables.com/model/394395-bambu-studio-orcaslicer-softfever-profiles-for-x1-

**Comment by nickyt1** [comment]
Sorry should have been more specific, looking for esun profiles for the anycubic vyper and creality cr10s pro please.

**Comment by andy-81** [comment]
Is there an easy way to copy another printer's filament profile between printers?

I have managed to work out how to create a new printer and am looking to use some of the filament profiles from a similar printer to start tuning from (rather than starting from scratch). Rather than manually opening and copying the data across is there an easy option to pull the filaments from another printer in and then I can adjust from there?

As an example, I can see eSUN filaments in the Bambu printer list, I would like to copy them into the Creality printers to then tune each one from there. I tried copying the files and editing the json files by adding the printer to the list, and then added the filament into the creality profile json file but was still unable to get it to allow me to select.

Thanks for any advise on an easy way to copy others profiles to then tweak

**Reply by little-big-h** [reply]
I also have this question. Given the current Bambu-lock-in-situation, I imagine that more people will be looking into alternatives (in my case, a Creality K1C). Having more profiles to kick off tuning would be a great help.
